### PR TITLE
Correct title bar offset for content interaction [DISCL-306]

### PR DESCRIPTION
**Pull request by rdumusc**

### CODE REVIEW

File: apps/DisplayCluster/resources/MasterContentWindow.qml [hunk 1]
```diff
@@ -57,16 +57,23 @@ BaseContentWindow {
         width: parent.width
         height: parent.height - (titleBar.visible ? titleBar.height : 0)
 
+        function getOffset(position) {
```
Comment (tribal-tec) [inline apps/DisplayCluster/resources/MasterContentWindow.qml]: does not return an offset, it removes it. getTouchPoint() or similar?

Comment (rdumusc) [inline apps/DisplayCluster/resources/MasterContentWindow.qml]: yes. removeOffset?

Comment (tribal-tec) [inline apps/DisplayCluster/resources/MasterContentWindow.qml]: Ok

**Comment by tribal-tec** [comment]
+1

**Pull request merged**